greynoise_connector.py

```diff
@@ -605,21 +634,48 @@ def _on_poll(self, param):
                 phantom.APP_ERROR, "Default on poll query unchanged"
             )
 
-        ret_val, data = self._gnql_query(
-            param, is_poll=True, action_result=action_result
-        )
-
-        if phantom.is_fail(ret_val):
+        query_results = self._gnql_query(param, is_poll=True, action_result=action_result)
+        if isinstance(query_results, bool) and phantom.is_fail(query_results):
             return action_result.get_status()
 
-        ret_val = self._process_query(data)
+        if not query_results:
+            return action_result.set_status(phantom.APP_SUCCESS, "No Data Found")
 
-        if phantom.is_fail(ret_val):
-            return action_result.set_status(
-                phantom.APP_ERROR, "Failed to process the query"
-            )
-        else:
-            return action_result.set_status(phantom.APP_SUCCESS)
+        self.save_progress("Creating containers")
+        for i in query_results[0][-int(param["size"]):]:
```

Comment:
`i` isn't an index here (which is what `i`,`j`,`k`, etc. is conventionally used for), can we name this something more descriptive or at least `result`?

Reply:
Done